neo/Ledger/Blockchain.cs

```diff
@@ -253,6 +255,15 @@ private void AddUnverifiedBlockToCache(Block block)
             blocks.AddLast(block);
         }
 
+        private void OnFillMemoryPool(IEnumerable<Transaction> transactions)
+        {
+            //TODO: Fill the memory pool with the transactions from the consensus context
+            // Step 1: Remove all the transactions in the memory pool
+            // Step 2: Add the transactions from the consensus context to the memory pool
+            // Step 3: Add the original transactions to the memory pool
```

Comment:
@jsolman Can you implement it? I don't know more about `MemoryPool` than you.

Reply:
I will implement it.

Reply:
I did a minor optimization of just invalidating transactions instead of removing, so step 3 goes away and the original transactions don't have to be re-added to the memory pool. Take a look.